apps/astarte_housekeeping/config/config.exs

```diff
@@ -2,4 +2,9 @@
 # and its dependencies with the aid of the Mix.Config module.
 import Config
 
+:persistent_term.put(
+  "customer_keyspace",
+  (System.get_env("keyspace") || "default") |> String.slice(0..39)
```

Comment:
Since we're using Skogsra to handle env vars (see https://github.com/astarte-platform/astarte/blob/master/apps/astarte_housekeeping/lib/astarte_housekeeping/config.ex), let's use it for `"keyspace"` too.
Also, I think that calling this value `customer_keyspace` might be inaccurate, as every Astarte realm is mapped to a Cassandra keyspace. UUIDv5 uses `namespace` to refer to the fixed part to add to some data, also `instance_id` could be a good choice (as the issue arises only when multiple Astarte instances share the same database)